targets/TARGET_NUVOTON/TARGET_M2351/TARGET_M23_NS/TARGET_NU_PFM_M2351_NPSA_NS/TARGET_NU_PREBUILD_SECURE/README.md

```diff
@@ -19,13 +20,51 @@ The pre-built secure code has the following hardware partition:
     - **PDMA0** hardwired to secure. Implements secure asynchronous transfer.
     - **PDMA1** configured to nonsecure. Implements nonsecure asynchronous transfer.
 
-## Pre-built secure code files
+In this memory partition, secure program is the most simplified.
+Then non-secure program can make use of the most memory available for its large application like pelion.
+
+To regenerate the default secure image/lib, in [non-PSA secure program template](https://github.com/OpenNuvoton/NuMaker-mbed-TZ-secure-example), configure **mbed_app.json**, where `"target.boot-stack-size": "0x600"` is to fit into just 8KiB secure SRAM:
+
+**mbed_app.json**:
+```
+{
+    "config": {
+        "tz-start-ns": {
+            "help": "Start address of TrustZone non-secure application"
+        }
+    },
+    "target_overrides": {
+        "*": {
+            "platform.stdio-baud-rate": 115200,
+            "platform.stdio-convert-newlines": true
+        },
+        "NU_PFM_M2351_NPSA_S": {
+            "target.device_has_remove": ["SERIAL", "SERIAL_ASYNCH", "SERIAL_FC", "STDIO_MESSAGES"],
+
+            "target.mbed_rom_start"             : "0x0",
+            "target.mbed_rom_size"              : "0x10000",
+            "target.mbed_ram_start"             : "0x20000000",
+            "target.mbed_ram_size"              : "0x2000",
+            
+            "target.boot-stack-size"            : "0x600"
+        }
+    }
+}
+```
```

Comment:
Why is it necessary to change mbed_app.json to build the default image? Shouldn't the default targets.json for NU_PFM_M2351_NPSA_S be modified to have these values as the default instead?

Reply:
Modify **README.md** and [Non-PSA secure code](https://github.com/OpenNuvoton/NuMaker-mbed-TZ-secure-example). The default secure program can re-generate without modification.